### ISSUE TITLE
Comparaison against DEIM and D-FINE detectors

### ISSUE THREAD

**Issue by tcourat** [sunsmarterjie/yolov12#4]
Hi,

Have you done a comparison with more recent DETR detectors such as [D-FINE](https://github.com/Peterande/D-FINE) and [DEIM ](https://github.com/ShihuaHuang95/DEIM)? 
On paper their result seem very solid and maybe better compared to yours, but I am unsure if they have the same experimental setup (same validation set/same image size...).

**Comment by sunsmarterjie** [comment]
Thanks for your suggestion. These detectors you mentioned are all great detectors! However, I think these methods all use pre-trained models. Additionally, I believe YOLOv12 still has the potential to achieve better performance and I am working on it.
